### PR TITLE
ref: Refactor `useConfig`

### LINKED ISSUE #237: Move `useOldConfig` into `utils`

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Pulls `useOldConfig` into a utility file and use throughput the app
- Removes env file and scripts for `turk-prolific`
- Removes `ReactDOMRender` in favor of `createRoot` (error from React 18)
- Render index in `<StrictMode>`

### CODE REVIEW

File: src/JsPsych/trials/examples/partials/photodiode.js [hunk 1]
```diff
@@ -33,7 +18,7 @@ try {
   }
 } catch (e) {
   console.warn('window.require is not available');
-  console.error(e);
+  // console.error(e);
```
Comment (eldu) [inline src/JsPsych/trials/examples/partials/photodiode.js]: uncomment or delete?

File: src/components/Login.jsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ function Login({
       </Form.Group>
       <Button
         className='width-50'
-        block
+        block='true'
```
Comment (eldu) [inline src/components/Login.jsx]: the string true?

**Review by eldu — APPROVED** [review]
I added in two comments, this looks great though !

File: src/components/Login.jsx (hunk 2 above)
Comment (RobertGemmaJr) [reply src/components/Login.jsx]: Was getting a weird error in the console about it not supposed to be a boolean... just removed it and it seems to be working fine though? I'll leave it deleted

**Review by hetd54 — APPROVED** [review]

**Pull request merged**